py/tests/plane_hog.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python3
+
+import pykms
+import time
+import random
+
+card = pykms.Card()
+res = pykms.ResourceManager(card)
+conn = res.reserve_connector()
+crtc = res.reserve_crtc(conn)
+
+mode = conn.get_default_mode()
+modeb = mode.to_blob(card)
+
+fb = pykms.DumbFramebuffer(card, mode.hdisplay >> 1, mode.vdisplay >> 1, "AR24");
+pykms.draw_test_pattern(fb);
+
+# Disable request
+card.disable_planes()
+
+plane_list = []
+
+while True:
+     plane = res.reserve_plane(crtc)
+     if plane:
+         print("Got plane %d" % plane.id)
+         plane_list.append(plane)
+     else:
+         break
+
+print("Got %d planes" % len(plane_list))
+
+req = pykms.AtomicReq(card)
+req.add(conn, "CRTC_ID", crtc.id)
+req.add(crtc, {"ACTIVE": 1,
+               "MODE_ID": modeb.id})
+
+input("Press enter to enable crtc %d" % crtc.id)
+r = req.commit_sync(allow_modeset = True)
```

Comment:
Move the atomic req creation and setup after the input(). It doesn't really help anything to split it and the commit like this.